### ISSUE TITLE
Remove DockerExec, it's Dead Code once DockerHealthCheck is Finished

### ISSUE THREAD

**Issue by original-brownbear** [yegor256/rultor#1065]
# Background

At this point `com.rultor.agents.docker.DockerExec` only has one purpose,
it periodically connects to the EC2 instance and by following functionality indicates whether the
service is up and running or not.
It's documented purpose `Remove old images from Docker.` is not fulfilled by it. The class was an attempt to resolve #906 . Eventually it turned out that the approach taken by the bash script that ships with this class `com/rultor/agents/docker/rmi.sh` caused certain issues described in #942 .
So in 7c874cb7cd06360fcab58c8c23ecff4e27657676 the script was quickly turned into dead code by commenting the actual logic in it.
Also the code at present is pretty much an 80% duplicate of it's successor `DockerHealthCheck`.
=> Fixing the doc on it to describe actual behaviour makes no sense anymore => this task.

# Task Requires

* remove `DockerExec` from the codebase
* remove `com/rultor/agents/docker/rmi.sh`
* ensure that this does not break the Service health display on the Rultor.com home page.
 * See stacktrace in #1063 for details on how this class currently throws an exception and hence turns the service health display red.

**Comment by original-brownbear** [comment]
@alex-palevsky this is a bug.

**Comment by original-brownbear** [comment]
@alex-palevsky this dependes on #1063.

**Comment by original-brownbear** [comment]
@alex-palevsky this depends on #1057.

**Comment by alex-palevsky** [comment]
> @alex-palevsky this is a bug.

@original-brownbear I just added bug tag here

**Comment by alex-palevsky** [comment]
@original-brownbear milestone set to `2.0` (correct me if I am wrong)

**Comment by alex-palevsky** [comment]
> @alex-palevsky this depends on #1057.

@original-brownbear of course, let's wait for #1057

**Comment by alex-palevsky** [comment]
@original-brownbear thanks for the report, I topped your acc for 30 mins, payment ID `AP-0C298578R5425400E`

**Comment by original-brownbear** [comment]
@alex-palevsky this is postponed.

**Comment by alex-palevsky** [comment]
> @alex-palevsky this is postponed.

@original-brownbear thanks, I added "postponed" label

**Comment by alex-palevsky** [comment]
> @alex-palevsky this is postponed.

@original-brownbear no problem, I will try to find somebody else

**Comment by alex-palevsky** [comment]
@original-brownbear we were waiting for #1057 - it is closed already
